### ISSUE TITLE
terraform plan failed after upgrade

### ISSUE THREAD

**Issue by hosmelq** [Qovery/terraform-provider-qovery#242]
After upgrading to the latest version (0.17.1) `terraform plan` starts failing with the following message.

```sh
Planning failed. Terraform encountered an error while generating this plan.
│
│ Error: Provider produced invalid plan
│
│ Provider "registry.terraform.io/qovery/qovery" planned an invalid value for
│ module.environment["Staging"].qovery_application.app.secrets: count in plan (cty.UnknownVal(cty.Number)) disagrees with count in
│ config (cty.NumberIntVal(8)).
│
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
│
│
│ Error: Provider produced invalid plan
│
│ Provider "registry.terraform.io/qovery/qovery" planned an invalid value for
│ module.environment["Staging"].qovery_application.app.environment_variables: count in plan (cty.UnknownVal(cty.Number)) disagrees with
│ count in config (cty.NumberIntVal(8)).
│
│ This is a bug in the provider, which should be reported in the provider's own issue tracker.
│
```

**Comment by mzottolaqovery** [comment]
Hello @hosmelq,
I'm looking at your issue. 
Out of curiosity, with which previous version did it work before upgrading to the `0.17.1` ?

**Comment by mzottolaqovery** [comment]
Can you also share the qovery console link of your application impacted ?

**Comment by hosmelq** [comment]
I was in ~> 0.16.5.

https://console.qovery.com/organization/16ea9b02-bd23-42bc-b21a-a489cf688dc4/project/1756ff01-6661-4147-9405-719a37b12bb2/environments/general

**Comment by mzottolaqovery** [comment]
Which version of terraform are you using ?

It seems to be a behavior present in the `1.5.0` that has been fixed recently in the `1.5.1`: https://github.com/hashicorp/terraform/issues/33371
I reproduced the same issue you had with the `1.5.0` and using the `1.5.1` fixed it.

Can you try on your side ?
